api/pacman-api-admin/src/main/java/com/tmobile/pacman/api/admin/repository/service/CreateAssetGroupServiceImpl.java

```diff
@@ -169,6 +178,20 @@ private List<DataSourceObj> getConfig(Set<String> dataSourceLs, List<HashMap<Str
         return dataSourceObjList;
     }
 
+    private Map<String, Set<String>> getAccIdForConfiguredSources() {
+
+        String query = "select accountId, platform from cf_Accounts  where platform in (select distinct dataSourceName " +
+                " from cf_Target) and accountStatus= 'configured'";
```

Comment:
Will it be easy to convert into JPA?
We already have classes for Accounts and Targets.

Reply:
yes, moved it to JPA. Good suggestion!